Translations/translation_FR.json

```
@@ -107,22 +107,22 @@
   },
   "menuValues": {
     "USBPDModeDefault": {
-      "displayText": "Default\nMode"
+      "displayText": "Mode\npar\nDéfaut"
```

The OLED is only capable of displaying 2 lines at the same time.
```suggestion
      "displayText": "Mode\npar Défaut"
```

---

No problem!
You are welcome. 😊